PythonVisualizations/AdvancedSorting.py

```diff
@@ -0,0 +1,570 @@
+import random
+import time
+from tkinter import *
+try:
+    from drawable import *
+    from VisualizationApp import *
+    from SortingBase import *    
+except ModuleNotFoundError:
+    from .drawable import *
+    from .VisualizationApp import *
+    from .SortingBase import *    
+      
+
+class AdvancedArraySort(SortingBase):
+
+    def __init__(self, size=10, title="Advanced Sorting", **kwargs):
+        super().__init__(title=title, **kwargs)
+        
+        self.buttons = self.makeButtons() 

```

Comment:
The initialization of the contents of the array should go here; not in the `SortingBase` class constructor.

Reply:
Implemented in commit 9aef402